src/definitions/Metadata.ts

```diff
@@ -54,7 +57,17 @@ export class FomodInfo extends XmlRepresentation<boolean> {
             if (value === undefined) continue;
 
             const child = document.createElement(key);
-            child.textContent = value;
+
+            if (Array.isArray(value)) {
+                for (const subChildValue of value) {
+                    // Append "element" tag to child, with the value
+                    const subChild = document.createElement('element');
+                    subChild.textContent = subChildValue;
+                    child.appendChild(subChild);
+                }
+            } else {
+                child.textContent = value;
+            }
```

Comment:
It may be worth investigating how existing tooling and mod managers handle multi-child elements in Info.xml. I have little knowledge on the subject personally, though the Fomod Creation Tool may be a good place to look if memory serves. I suspect we'll end up creating a class for more complex representations sometime down the line, even if just to handle the element name.